tests/ui/static/static-closures-with-nonstatic-return.rs

```diff
@@ -0,0 +1,15 @@
+// check-pass
```

Comment:
Let's also move this to `tests/ui/closures`, `static` is for `static` items, I think.

Reply:
Aha, moved to `tests/ui/closures`.